tests/acceptance/filesForUpload/lorem.txt

```diff
@@ -0,0 +1,7 @@
+Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium,
+totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae
+dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit,
+sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est
+qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora
+incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad minima veniam, quis nostrum
+exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur?
```

Comment:
may be this file can be obtain from `core`?

Reply:
yes, we can. Thanks for reminder

Reply:
I think this file can be removed now.

Reply:
@grgprarup yes thanks missed that!

Reply:
@SagarGi @grgprarup It would have been nice to use the file from core but turns out that the function that is doing the upload is used to upload the files present in the local ocis repo and the tests failed in https://drone.owncloud.com/owncloud/ocis/14967/41/8, while we *can* upload these files in core and then use it from there but I'm going to keep them here and add a `lorem.txt` file here and use it from here

Reply:
Used the already present file. so that's better I think.